### DISCUSSION TITLE
What does the box.raw data mean?

### DISCUSSION THREAD

**Discussion by zcrwind** [deepmodeling/deepmd-kit#13] [Q&A] [answered]
In the readme, the authors  say that the `box.raw` data should be prepared. However, I cannnot find the specific meaning of the `box.raw` data in neither paper or readme. Can someone help me with this? Many thanks.

**Comment by jameswind** [comment, marked as answer]
It is in the Readme: "... In box.raw, the 9 components of the box vectors
should be provided on each line ..."

**Reply by yanggthomas** [reply]
hi, does this mean that deepmd is supposing triclinic coordinates by default

**Reply by wanghan-iapcm** [reply]
yes
